src/Monitor/Monitor/help/Add-AzMetricAlertRuleV2.md

```diff
@@ -0,0 +1,417 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll-Help.xml
+Module Name: Az.Monitor
+online version:
+schema: 2.0.0
+---
+
+# Add-AzMetricAlertRuleV2
+
+## SYNOPSIS
+Adds or updates a V2 (non-classic) metric-based alert rule.
+
+## SYNTAX
+
+### CreateAlertByResourceId
+For creating metric alert rule targeted on a single resource
```

Comment:
@AnanyShah was this added manually? If so, please remove the descriptions under each parameter set name